packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -33,9 +33,9 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
     private lineChartRef: any;
     private mouseIsDown = false;
     private positionZoomMouse = 0;
-    private positionZommkeyboard = 0;
+    private positionZomCenter = 0;
```

Comment:
Zom? "positionZomCenter"

Reply:
this is the position from where I want to zoom . Because when i zoom with "+" is not the same position when i zoom with "w".
The zoom with "+" follow the center of the view and the zoom with " w" it follows the mouse cursor

Reply:
I mean why is it "Zom" rather than "Zoom"?

Reply:
ah sorry thank you i didn't notice